docker/package/model.py

```diff
@@ -313,7 +313,7 @@ def gen_control_file(self, build_deps, run_deps, ubuntu_version, out):
 Section: utils
 Priority: optional
 Maintainer: {self.meta.maintainer}
-Build-Depends: debhelper (>=9), {"dh-systemd (>= 1.5), " if ubuntu_version != "jammy" else ""} autotools-dev, {str_build_deps}
```

Comment:
Is it right that we don't need this workaround for `noble`?

Reply:
Yes, it is needed only for focal